.gitignore

```diff
@@ -6,8 +6,8 @@
 /out-tsc
 
 # dependencies
-/code/node_modules
-/code/package-lock.json
+/web/node_modules
+/web/package-lock.json
```

Comment:
We should be committing the lock file as well right ? 

Otherwise how will you deal with issues where developers will work on latest versions and compain things not working.

Reply:
Ok sir, I will remove it from gitignore and update the PR. Actually, I was chaos about this file.